apps/backend/src/modules/contributors/contributor.repository.ts

```diff
@@ -57,57 +58,16 @@ class ContributorRepository implements Repository {
 
 	public async findAll({
 		contributorName,
-		hasHidden = true,
+		hasHidden = false,
 		page,
 		pageSize,
-	}: { hasHidden?: boolean } & ContributorGetAllQueryParameters): Promise<
+		projectId,
+	}: ContributorGetAllRequestDto): Promise<
 		PaginationResponseDto<ContributorEntity>
 	> {
 		const query = this.contributorModel
 			.query()
 			.orderBy("createdAt", SortType.DESCENDING)
```

Comment:
As you left only one method for retrieving contributors, I think we might also need to pass option for sorting or smth. When we're retrieving all contributors, they were ordered by the time they were created, and when we're retrieving them by projectId, they were ordered by the time of their last activity. (The last thing was merged to main yesterday) Or maybe we can sort all contributors by their last activity date.

Reply:
Oh, yeah, I see in the chat that you've noticed it already😅

Reply:
Yeah. I'm also thinking of sortBy option, cause in the table we need to sort by created_at (so rows don't swap while script updates analytics), and in the project details page we need to sort by last_activity_date.

Of course I may choose sorting depanding on `projectId` (either it was passed or not), but this is not a good solution, as in the table I also retrieve contributors by project id, and the issue will remain

>maybe we can sort all contributors by their last activity date.

Also thinking about this but not sure. I'll clarify this.

Thanks for suggession)

Reply:
I have added `orderBy` to the query